net/modemmanager/files/25-modemmanager-net

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# Copyright (C) 2016 Velocloud Inc
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+
+# Load common utilities
+. /etc/modemmanager/modemmanager.common
+
+# Always make sure the rundir exists
+mkdir -m 0755 -p "${MODEMMANAGER_RUNDIR}"
+
+# We require a interface name
+[ -n "${INTERFACE}" ] || return
```

Comment:
Here I'm a little bit unsure: could we move this check above before sourcing modemmanager.common? This could safe us from parsing it and safes a few CPU cycles when INTERFACE is not given... Also the return would exit the script, so other scripts use `exit` at similar points, so I would also prefer exit.